### DISCUSSION TITLE
Rule Proposal: [prefer-renderer] Use Renderer2 for supported DOM manipulations

### DISCUSSION THREAD

**Discussion by ansjllenasm** [angular-eslint/angular-eslint#2091] [New Rule Proposals]
Favor the usage of the `Renderer2 ` APIs for supported DOM manipulations to ensure compatibility and abstraction from direct DOM interactions.

**Comment by JamesHenry** [comment]
I would recommend using https://eslint.org/docs/latest/rules/no-restricted-imports#importnames for this purpose, there's no need for a super specific rule for this
